ckanext/vocabulary/blueprints/vocabulary.py

```diff
@@ -133,9 +129,31 @@ def delete_tags(vocab_id, id):
     return render('vocabulary/edit_tags.html', {"tag": tag, 'vocab_name': vocab_name})
 
 
+def delete(id):
+    context = {'model': model, 'session': model.Session,
+                'user': c.user, 'auth_user_obj': c.userobj}
+    if not c.userobj.sysadmin:
+        base.abort(403, _(u'Unauthorized to create new tags'))
```

Comment:
@steveoni  I think the error message should be for deleting tags rather than creating new tags. 
```suggestion
        base.abort(403, _(u'Unauthorized to manage tags'))
```